Enterprise-CMCS/cmcs-eregulations
# EREGCSC-2636 -- Remove "Uploaded By" label on Uploaded Documents

Resolves [EREGCSC-2636](https://jiraent.cms.gov/browse/EREGCSC-2636)

**Description**

On internal files, we display an "uploaded by xyz" message. This will soon be refactored to display information about visibility that will be styled and displayed in the same way, so for now we are simply commenting out this component where it is used, and a future story will refactor it to display "Visible to xyz" information instead.

It will also be informative if we remove this message and people miss it.

**This pull request changes:**

- Comments out `<DivisionLabel />` using HTML comment syntax: ``
- Updates snapshot to reflect this change 

**Steps to manually verify this change:**

1. Green check marks
2. Visit experimental deployment
3. Visit the admin panel and view Uploaded Documents.  You should see at least one doc.  Verify that the doc includes a Division and a Group.
4. Visit Subject page, filter by internal documents, and view existing document
5. Ensure that this document does not have an "Uploaded by xyz" label, even though the document has an attached Division and Group
6. Visit the Related Regulation Citation
7. Ensure that the uploaded document does not have an "Uploaded by xyz" label when viewed in the right sidebar
